linode/instance/schema_resource.go

```diff
@@ -454,6 +454,19 @@ var resourceSchema = map[string]*schema.Schema{
 		Description: "Fields related to the Placement Group this instance is assigned to.",
 		Optional:    true,
 		MaxItems:    1,
+		DiffSuppressFunc: func(k, oldValue, newValue string, d *schema.ResourceData) bool {
+			return d.Get("placement_group_externally_managed").(bool)
+		},
+	},
+	"placement_group_externally_managed": {
```

Comment:
Users can _technically_ achieve similar functionality by adding `placement_group` to the `ignore_changes` lifecycle policy but I figured a dedicated attribute would add some extra user confidence and give us a bit more flexibility.

Reply:
Hmm, good question! My only concern with making the `placement_group` computed would be that users would lose the ability to unassign their instance from the PG